src/main/java/ch/wisv/areafiftylan/service/EventServiceImpl.java

```diff
@@ -0,0 +1,122 @@
+package ch.wisv.areafiftylan.service;
+
+import ch.wisv.areafiftylan.dto.EventDTO;
+import ch.wisv.areafiftylan.exception.EventException;
+import ch.wisv.areafiftylan.model.Event;
+import ch.wisv.areafiftylan.model.Team;
+import ch.wisv.areafiftylan.service.repository.EventRepository;
+import com.google.common.base.Strings;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+
+import java.util.Collection;
+import java.util.Set;
+
+/**
+ * Created by Sille Kamoen on 27-3-16.
+ */
+@Service
+public class EventServiceImpl implements EventService {
+
+    private EventRepository eventRepository;
+
+    @Autowired
+    public EventServiceImpl(EventRepository eventRepository) {
+        this.eventRepository = eventRepository;
+    }
+
+    @Override
+    public void registerTeam(Long eventId, Team team) {
+        Event event = getEventById(eventId);
```

Comment:
Why does this method require the Team and not teamId? I personally believe not logic should be in the Controller other than Authorization and responses.